iothub_service_client/src/iothub_devicetwin.c

```diff
@@ -28,8 +28,10 @@
     IOTHUB_TWIN_REQUEST_UPDATE_DESIRED
 
 MU_DEFINE_ENUM(IOTHUB_TWIN_REQUEST_MODE, IOTHUB_TWIN_REQUEST_MODE_VALUES);
+
```

Comment:
So only public facing enums are getting the treatment?

Reply:
Yes.  I don't think the ROI is as good on internal logic.  At minimum I'd want to get the public API changes in now.

---
In reply to: [350385167](https://github.com/Azure/azure-iot-sdk-c/pull/1342#discussion_r350385167) [](ancestors = 350385167)